server/src/compression.rs

```diff
@@ -838,29 +830,21 @@ pub mod test_util {
                 }
                 values.append(&mut random);
             }
+
+            // Generates randomly selected constant data.
```

Comment:
Remove `randomly selected` since this just uses `10.0` now.

Reply:
Fixed.